src/App.js

```diff
@@ -98,12 +100,72 @@ function App() {
   function moveToHistory(input) {
     setHistoryItems([...historyItems, ...shoppingCard]);
     setShoppingCard([]);
+    window.confirm(
+      `Sehr geehrte/r ${inputData.name}! Danke für Ihren Kauf! Wir leiten Sie zu dem Kaufübersicht weiter.`
+    );
     navigate("/history");
   }
   function clearHistory() {
     setHistoryItems([]);
   }
+  function feedbackSubmit(user, feedback, elementForFeedback) {
+    const objectExists = allDataItems.find(
+      (object) => object.id === elementForFeedback.id
+    );
+    if (objectExists) {
+      setAllDataItems(
+        allDataItems.map((element) =>
+          element.id === elementForFeedback.id
+            ? {
+                ...elementForFeedback,
+                user: user,
+                feedback: feedback,
+              }
+            : element
+        )
+      );
+    } else {
+      return null;
+    }
 
+    let isExecuted = window.confirm(
+      "Danke für Ihre Bewertung! Nach Bestätigung werden Sie zum Artikel Details weitergeleitet!"
+    );
+    if (isExecuted) {
+      navigate(`/${elementForFeedback.id}`);
+      window.location.reload();
+    } else {
+      return null;
+    }
+    console.log(isExecuted);
```

Comment:
Please remove console logs for debugging

Reply:
done